config.yaml

```diff
@@ -24,9 +24,20 @@ database:
   type: static
 
 destinations:
-  - type: duckdb
+  - type: redshift
```

Comment:
Please do not modify the default config.yaml. This file is used as the default config if no credentials are provided.